shoop/admin/views/dashboard.py

```diff
@@ -22,3 +23,7 @@ def get_context_data(self, **kwargs):
             blocks.extend(module.get_dashboard_blocks(request=self.request))
         context["activity"] = get_activity(request=self.request)
         return context
+
+    def get(self, request, *args, **kwargs):
+        send_telemetry(request)
+        return super(DashboardView, self).get(request, *args, **kwargs)
```

Comment:
I would rather see this code here being more explicit about what's going to happend, e.g.
```python
def get(self, request, *args, **kwargs):
    if telemetry.should_send(request):
        telemetry.send(request)
    return super(DashboardView, self).get(request, *args, **kwargs)
```
That would make it more clear that we don't actually send telemetry for each request, and reader can see the `telemetry.should_send` function to see when it actually happens if he cares. (Feel free to ignore renaming the `telemetry.send_telemetry` to just `telemetry.send`... just a personal taste..)